File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] parts = binomial.split("\\s*[-+*]\\s*");
        int num1 = Integer.parseInt(parts[0]);
        int num2 = Integer.parseInt(parts[1]);

        if (binomial.contains("+")) return num1 + num2;
        if (binomial.contains("-")) return num1 - num2;
        if (binomial.contains("*")) return num1 * num2;

        return 0;
    }
}

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    const parts = binomial.split(" ");
    const num1 = Number.parseInt(parts[0]);
    const num2 = Number.parseInt(parts[2]);

    if (binomial.includes("+")) return num1 + num2;
    if (binomial.includes("-")) return num1 - num2;
    if (binomial.includes("*")) return num1 * num2;
    if (binomial.includes("*")) return num1 * num2;
}